corehq/apps/domain/models.py

```diff
@@ -681,7 +681,7 @@ def get_latest_app_id(doc_id):
                 if app:
                     return app._id, app.doc_type
 
-            for app in get_apps_in_domain(self.name):
+            for app in get_brief_apps_in_domain(self.name):
```

Comment:
It turns out this is the only place (besides tests) that actually wanted to call the brief version. By default full=False, but all other callers were relying on the behavior that made it equivalent to full=True. (I had to read the context of each call to figure that out.)